app/views/operatingsystems/_templates.html.erb

```diff
@@ -1,2 +1,3 @@
-<%= selectable_f f,:provisioning_template_id, @operatingsystem.provisioning_templates.where(:template_kind_id => f.object.template_kind_id).collect { |c| [c.name, c.id]}, { :include_blank => true }, { :label => _(f.object.template_kind.to_s) } %>
+<%= selectable_f f,:provisioning_template_id, @operatingsystem.provisioning_templates.where(:template_kind_id => f.object.template_kind_id).collect { |c| [c.name, c.id]}, { :include_blank => true }, { :label => _(f.object.template_kind.to_s), :label_help => _(f.object.template_kind.description) }  %>
+
```

Comment:
Nitpick: Two spaces at the end of the line 1, is the extra line needed? I am fine just checking.